### ISSUE TITLE
Odd click behaviour with slider

### ISSUE THREAD

**Issue by justinmclean** [apache/royale-asjs#102]
To reproduce use this code:
```
<?xml version="1.0" encoding="utf-8"?>
<js:Application xmlns:fx="http://ns.adobe.com/mxml/2009"
               xmlns:js="library://ns.apache.org/royale/basic">
   <js:valuesImpl>
       <js:SimpleCSSValuesImpl/>
   </js:valuesImpl>

    <js:beads>
        <js:ApplicationDataBinding />
    </js:beads>

   <js:initialView>
       <js:View width="200" height="200">
           <js:Container>
               <js:beads>
                   <js:VerticalLayout />
               </js:beads>
               <js:Slider id="s" minimum="0" maximum="1000" snapInterval="5" width="500" />
               <js:Label text="{s.value}" />
           </js:Container>
       </js:View>
   </js:initialView>
</js:Application>
```

Click on the middle of the track and the thumb will move there and have a value around 500, click again and the thumb moves right, click again and it will move right again. I would of expected to stay in the same place and not move right on any of those clicks.

**Comment by pentapache** [comment]
This should be fixed by commit 02f0dd4

**Comment by pentapache** [comment]
This should be fixed by commit 02f0dd4
